support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -78,7 +78,10 @@ export type Test = {
 	// before activating this test eg. '/(uk|us|au|ca|nz)/subscribe$'
 	targetPage?: string | RegExp;
 	omitCountries?: IsoCountry[];
-	excludeCountriesSubjectToVatCompliantAmounts: boolean;
+	// Some users will see a version of the checkout that only offers
+	// the option to make contributions. We won't want to include these
+	// users in some AB tests
+	excludeCountriesSubjectToContributionsOnlyAmounts: boolean;
```

Comment:
So this is generally `true` unless testing  the contibutions-only checkout

Reply:
Yes that's correct, I've made it a non-optional value so whoever's setting up a test will need to consider this